Resources/Prototypes/Entities/Objects/Decoration/flora.yml

```diff
@@ -414,7 +414,7 @@
       fix1:
         shape:
           !type:PhysShapeAabb
-          bounds: "-0.8,-1.5,0.8,-0.8"
+          bounds: "-0.5,0.75,0.5,0"
```

Comment:
Winding order is left-bottom-right-top so this is out of order.

Reply:
@metalgearsloth Apologies. Good now?

Reply:
yeah, also please avoid force-pushing and send screenshot of the new bounds.